### PR TITLE
[WIP] Revamp README

### PR DESCRIPTION
**Pull request by ghost**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes
| Fixed tickets | #21235 
| License       | MIT
| Doc PR        | N/A

This pull request revamps the `README.md` file based on all the feedback in the ticket. Currently, this PR is work in progress and should not be merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,3 @@
...
-
-[1]: https://symfony.com/doc/current/quick_tour/index.html
-[2]: https://symfony.com/doc/current/
-[3]: https://symfony.com/doc/current/contributing/code/index.html
-[4]: https://symfony.com/doc/current/contributing/code/patches.html#check-list
-[5]: https://symfony.com/doc/current/contributing/code/patches.html#make-a-pull-request
-[6]: https://symfony.com/doc/master/contributing/code/tests.html
-[7]: https://symfony.com/doc/current/book/installation.html#installing-the-symfony-installer
-[8]: https://symfony.com/doc/current/contributing/community/reviews.html
+<div align="center">
```
Comment (stof) [inline README.md]: using HTML is a bad idea. This is a markdown file

Comment (Ricknox) [inline README.md]: Yes, I knew it the moment I started this PR. I'll fix that.

Comment (Ricknox) [inline README.md]: Markdown doesn't care with positioning. It has to be done with HTML or the image could be changed to Markdown but then I purpose to use [this image](http://symfony.com/logos/symfony_black_02.svg?v=4) and just leave it at the left.

**Comment by Ricknox** [comment]
I'll leave this PR stalled for now since @javiereguiluz added a comment [in the issue](https://github.com/symfony/symfony/issues/21235#issuecomment-272428720) about approval from the symfony deciders. Additional work would destroy the time and effort.

**Comment by javiereguiluz** [comment]
@Ricknox thanks! I'm going to lock this PR too until the related issue is approved to avoid duplicated comments.

**Comment by fabpot** [comment]
Let's close this PR until we know what to do.

**Pull request closed without merging**